src/cryptoadvance/specter/server_endpoints/nodes.py

```diff
@@ -0,0 +1,333 @@
+        node_alias=node_alias,
+        test=test,
+        specter=app.specter,
+        rand=rand,
+    )
+
+
+@nodes_endpoint.route("specter_node/<node_alias>/", methods=["GET", "POST"])
+@login_required
+def internal_node_settings(node_alias):
+    err = None
+    if node_alias:
+        try:
+            node = app.specter.node_manager.get_by_alias(node_alias)
+            if node.external_node:
+                return redirect(
+                    url_for(
+                        "nodes_endpoint.node_settings",
+                        node_alias=node.alias,
+                    )
+                )
+        except:
+            return render_template(
+                "base.jinja", error="Node not found", specter=app.specter, rand=rand
+            )
+    else:
+        # TODO: Allow internal node setup here?
+        return redirect(
+            url_for(
+                "nodes_endpoint.internal_node_settings",
+                node_alias=node.alias,
+            )
+        )
+
+    if not current_user.is_admin:
+        flash("Only an admin is allowed to access this page.", "error")
+        return redirect("")
+    # The node might have been down but is now up again
+    # (and the checker did not realized yet) and the user clicked "Configure Node"
+    if node.rpc is None:
+        node.update_rpc()
+
+    err = None
+    if request.method == "POST":
+        action = request.form["action"]
+
+        if action == "rename":
+            node_name = request.form["newtitle"]
+            if not node_name:
+                flash("Node name must not be empty", "error")
+            elif node_name == node.name:
+                pass
+            elif node_name in app.specter.device_manager.devices_names:
+                flash("Node with this name already exists", "error")
+            else:
+                node.rename(node_name)
+        elif action == "forget":
+            if not node_alias:
+                flash("Failed to deleted node. Node isn't saved", "error")
+            elif len(app.specter.node_manager.nodes) > 1:
+                app.specter.node_manager.delete_node(node, app.specter)
+                if bool(request.form.get("remove_datadir", False)):
+                    shutil.rmtree(os.path.expanduser(node.datadir), ignore_errors=True)
+                flash("Node deleted successfully")
+                return redirect(
+                    url_for(
+                        "nodes_endpoint.node_settings",
+                        node_alias=app.specter.node.alias,
+                    )
+                )
+            else:
+                flash(
+                    "Failed to deleted node. Specter must have at least one node configured",
+                    "error",
+                )
+        elif action == "stopbitcoind":
+            try:
+                node.stop()
+                time.sleep(5)
+                flash("Specter stopped Bitcoin Core successfully")
+            except Exception:
```

Comment:
Why do you catch Exception here? Can you either remove it or catch a more specific Exception?
If that's difficult for some reason, at least add a `logger.exception(e)`